### ISSUE TITLE
[Feature Request] Scrollbar for Animation Timeline

### ISSUE THREAD

**Issue by helloar14** [Inochi2D/inochi-creator#227]
### Validations

- [X] I have checked for similar feature requests and could not find any.
- [X] I have made sure this is not an already-existing feature.

### Description

(I know about the plan for big UI overhaul so just set aside this as a note for the future) Currently the way to navigate timeline is by mouse scroll plus modifier key (shift/alt). An actual scrollbar would be nice.

### Suggested solution

- Add scrollbar for animation timeline.
- For keyboard navigation, maybe show the instructions in the bottom status bar like how it's already been done in path deformation mode, for example.

**Comment by LunaTheFoxgirl** [comment]
Yeah, with how the timeline is currently implemented this would be a funky undertaking. So for the time being you'll have to use your scroll wheel...
